api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -110,6 +113,52 @@ void shouldProcessClaimFromPregnancyToChildTurningFour() throws JsonProcessingEx
         assertThat(claim.getCardStatusTimestamp()).isAfterOrEqualTo(now);
     }
 
+    /**
+     * Run through the lifecycle of a claim where a claimant becomes pregnant but no children ever appear on the feed.
+     *                                                     | one year
+     * |...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|...|
+     * | claim starts (due date in 25 weeks)
+     * |........................| Due date
+     * |...............................| email asking claimant to tell their benefit agency about new child
+     * |.......................................| email about coming off the scheme
+     * |.......................................................| email that the card will be cancelled in one week
+     */
+    @Test
+    @Disabled("HTBHF-2377")
+    @SuppressWarnings("checkstyle:VariableDeclarationUsageDistance")
+    public void shouldProcessClaimFromPregnancyToComingOffTheSchemeWithNoChildrenFromPregnancy() throws JsonProcessingException, NotificationClientException {
```

Comment:
I don't see anything in this test which verifies that the new email is sent.

Reply:
Line 138, assertThatReportABirthReminderEmailWasSent(claim);